phases/19-capstone-projects/38-classifier-finetuning/code/tests/test_main.py

```
@@ -0,0 +1,195 @@
+
+
+class DatasetTests(unittest.TestCase):
+    def test_make_dataset_balanced(self) -> None:
+        texts, labels = make_dataset(n_per_class=50, seed=1)
+        self.assertEqual(len(texts), 100)
+        self.assertEqual(labels.count(0), 50)
+        self.assertEqual(labels.count(1), 50)
+
+    def test_stratified_split_preserves_balance(self) -> None:
+        texts, labels = make_dataset(n_per_class=100, seed=2)
+        tr_t, tr_y, te_t, te_y = stratified_split(texts, labels, test_frac=0.2, seed=2)
```

Ruff (0.15.14)

[warning] Unpacked variable `tr_y` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)